Tests/LibELF/CMakeLists.txt

```diff
@@ -1,12 +1,30 @@
 set(CMAKE_SKIP_RPATH FALSE)
+if (NOT BUILD_LAGOM)
```

Comment:
Do you really want to check for a Lagom build here, or for an if (SERENITYOS) os-check?

Reply:
Yes, because I want correct rpaths in Build/lagom directory on Lagom even if Serenity is self-hosted